include/boost/spirit/home/x3/support/traits/container_traits.hpp

```diff
@@ -180,25 +165,25 @@ namespace boost { namespace spirit { namespace x3 { namespace traits
     struct append_container
     {
         template <typename Iterator>
-        static void reserve(Container& c, Iterator first, Iterator last, std::false_type)
```

Comment:
Prefer std instead of Boost, unless there's some performance benefit. In this case, it might be the other way round. MPL has some extra scaffoldings that add to the total compile time.